maintainers/maintainer-list.nix

```diff
@@ -7217,6 +7217,12 @@
     githubId = 1267527;
     name = "Daniel Firth";
   };
+  lodi = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
Okay, I split it into its own commit.  I tried to insert myself as alphabetically as possible but the file is already somewhat loosely sorted.